.github/workflows/ci.yaml

```
@@ -1,40 +1,62 @@
+        uses: dtolnay/rust-toolchain@stable
         with:
-          toolchain: stable
-          override: true
-      - name: Install moonbit
+          components: rustfmt, clippy
+      - name: Setup Rust
+        if: runner.os == 'Windows'
+        run: |
+          rustup toolchain install stable --profile default --no-self-update
+          rustup default stable
+          rustup target add wasm32-wasip1          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)